nms/app/packages/magmalte/grafana/dashboards/AnalyticsDashboards.js

```diff
@@ -15,153 +15,158 @@
-              legendFormat: '{{apn}}',
-            },
-          ],
-          unit: 'Bps',
-          description:
-            'Average user upload throughput for a given APN in the last N days',
-        },
-        {
-          title: 'Throughput Per APN (Download)',
-          targets: [
-            {
-              expr: 'throughput_per_ap{direction="in",apn=~"$apn"}',
-              legendFormat: '{{apn}}',
-            },
-          ],
-          unit: 'Bps',
-          description:
-            'Average user download throughput for a given APN in the last N days',
-        },
-      ],
-    },
-    {
-      title: 'User Consumption',
-      panels: [
-        {
-          title: 'User Consumption (Upload)',
-          targets: [
-            {
-              expr: 'user_consumption{direction="out",networkID=~"$networkID"}',
-              legendFormat: '{{networkID}} - Days: {{days}}',
-            },
-          ],
-          unit: 'decbytes',
-          description:
-            'Total user upload consumption over the network in the past N days.',
-        },
-        {
-          title: 'User Consumption (Download)',
-          targets: [
-            {
-              expr: 'user_consumption{direction="in",networkID=~"$networkID"}',
-              legendFormat: '{{networkID}} - Days: {{days}}',
-            },
-          ],
-          unit: 'decbytes',
-          description:
-            'Total user download consumption over the network in the past N days.',
-        },
-        {
-          title: 'User Consumption Hourly (Upload)',
-          targets: [
-            {
-              expr:
-                'user_consumption_hourly{direction="out",networkID=~"$networkID"}',
-              legendFormat: '{{networkID}}',
-            },
-          ],
-          unit: 'decbytes',
-          description:
-            'Total user upload consumption over the network in the past hour.',
-        },
-        {
-          title: 'User Consumption Hourly (Download)',
-          targets: [
-            {
-              expr:
-                'user_consumption_hourly{direction="in",networkID=~"$networkID"}',
-              legendFormat: '{{networkID}}',
-            },
-          ],
-          unit: 'decbytes',
-          description:
-            'Total user download consumption over the network in the past hour.',
-        },
-      ],
-    },
-    {
-      title: 'Authentications',
-      panels: [
-        {
-          title: 'Authentications Over Time',
-          targets: [
-            {
-              expr: 'authentications_over_time{networkID=~"$networkID"}',
-              legendFormat: '{{networkID}} - Code: {{code}}',
-            },
-          ],
-          description:
-            'Total number of authentication failures or successes in the network in the last N days.',
-        },
-      ],
-    },
-  ],
+export const AnalyticsDBData: GrafanaDBData = (networkIDs: Array<string>) => {
```

Comment:
You can ignore the changes to all the dashboard data, it all just had to be indented one tab and so the line count is super inflated